<?xml version="1.0"?>
<launch>
    <!-- param -->
    <arg name="robot_param" default="$(find point_follow_planner)/config/robot_param.yaml"/>
    <arg name="hz" default="5"/>
    <!-- topic name -->
    <arg name="robot_footprint" default="/robot_footprint"/>

    <node pkg="point_follow_planner" type="robot_footprint_publisher" name="robot_footprint_publisher" output="screen">
        <rosparam command="load" file="$(arg robot_param)"/>
        <param name="hz" value="$(arg hz)"/>
        <remap from="/robot_footprint" to="$(arg robot_footprint)"/>
    </node>
</launch>
